lamindb_setup/core/_settings_storage.py

```diff
@@ -92,6 +98,10 @@ def init_storage(storage: UPathStr, region: Optional[str] = None) -> "StorageSet
         ssettings._description = f"Created as default storage for instance {uid}"
         ssettings._uuid = init_storage_hub(ssettings)
         logger.important(f"registered storage: {ssettings.root_as_str}")
+    if root.startswith("s3://"):
+        # we need to touch a 0-byte object in the storage location to avoid
```

Comment:
This comment should be with the mark_storage_root function, then we don't need this comment twice in the code base.